tfhe/benches/boolean/bench.rs

```diff
@@ -17,7 +17,7 @@ criterion_group!(
 criterion_main!(gates_benches);
 
 /// Helper function to write boolean benchmarks parameters to disk in JSON format.
-pub fn write_to_json_boolean<T: Into<CryptoParametersRecord>>(
+pub fn write_to_json_boolean<T: Into<CryptoParametersRecord<u64>>>(
```

Comment:
what's the u64 here ? if it's for the ciphertext modulus I believe boolean is u32 ?